.changeset/afraid-coins-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+"@commercetools-docs/gatsby-theme-docs": patch
```

Comment:
In theory we don't need a changeset here, nothing in the code changed that would need a release.